pca-main.template

```diff
@@ -395,8 +395,8 @@ Parameters:
     
   DatabaseName:
     Type: String
-    Default: 'pca'
```

Comment:
Rather than force user to enter a unique database name, can we instead allow an empty default and use a condition in the template to replace an empty DatabaseName with a unique name constructed from the StackName?